packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -166,6 +170,7 @@ export const Webchat = forwardRef((props, ref) => {
   const { initialSession, initialDevSettings, onStateChange } = props
   const [botonicState, saveState, deleteState] = useLocalStorage('botonicState')
   const deviceAdapter = new DeviceAdapter()
+  const [coverComponentIsOpened, setCoverComponentIsOpened] = useState(false)
```

Comment:
I think it would be better to have this two methods in our reducers files, just like we did for persistentMenu or emojis